stores/autopilot_test.go

```diff
@@ -11,13 +11,11 @@ import (
 )
 
 func TestAutopilotStore(t *testing.T) {
-	db, _, _, _, err := newTestSQLStore(t.TempDir())
-	if err != nil {
-		t.Fatal(err)
-	}
+	ss := newTestSQLStore(t, defaultTestSQLStoreConfig)
```

Comment:
This little refactor is separate from metrics right? Just a thought but if you were to separate it the overall metrics PR would probably be quite a bit smaller and touch less files, seems like this change is contributing to a large part of the dif.

Reply:
https://github.com/SiaFoundation/renterd/pull/678